src/itemset.js

```diff
@@ -203,30 +211,51 @@ define([
     }
 
     function itemsetWidget(mug, options) {
-        var widget = datasources.dataSourceWidget(mug, options, "Data Source"),
+        var widget = datasources.fixtureWidget(mug, options, "Data Source"),
             super_getUIElement = widget.getUIElement,
             super_getValue = widget.getValue,
             super_setValue = widget.setValue,
-            handleChange = widget.handleChange.bind(widget),
-            labelRef = refSelect("label_ref", "Choice Label", widget.isDisabled()),
-            valueRef = refSelect("value_ref", "Choice Value", widget.isDisabled());
+            super_handleChange = widget.handleChange.bind(widget),
+            labelRef = refSelect("label_ref", "Choice Label", false),
+            valueRef = refSelect("value_ref", "Choice Value", false),
+            filterRef = refSelect("filter_ref", "Choice Filter", false);
```

Comment:
Will need to change this over to a proper widget to get the edit button (I think)